internal/ogc/tiles/main_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var tilePath = "/foo/12/34/56"
```

Comment:
dekken we met deze testen zowel de dataset als geodata/collection tiles healthcheck logica?